sale_variant_configurator/models/sale_order.py

```diff
@@ -50,6 +55,27 @@ class SaleOrderLine(models.Model):
         ),
     ]
 
+    @api.depends("product_tmpl_id")
+    def _compute_product_uom(self):
+        lines_with_template = self.filtered(
+            lambda x: x.product_tmpl_id and not x.product_id
+        )
+        for line in lines_with_template:
+            if not line.product_uom or (
+                line.product_tmpl_id.uom_id.id != line.product_uom.id
+            ):
```

Comment:
Is this comparison due to any reason? If so, please put a comment saying the reason for not being removed in the future.

Reply:
Added comment to clarify.